content/millennium/dstu2/proprietary-codes.md

```diff
@@ -25,8 +25,9 @@ The following are true for all Millennium proprietary codes:
 * The `code` value is the numeric code value as a string
 * The `display` value is the code value display
 * The `userSelected` value is set to true
+* On inbound requests (POST, PUT, PATCH), proprietary codes and standard codes may not be set in the same coding array. If they are, the create or update request will fail with a 422 Unprocessable Entity response.
```

Comment:
wrap http codes in code ticks

```
`422 Unprocessable Entity`
```

Reply:
https://github.com/cerner/fhir.cerner.com/pull/313/commits/3f8be98a496338484e737cd857ebb0d7095f3bce